tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -208,6 +210,13 @@ final class BuildRunner extends Runner {
   /// The [Build] to run.
   final Build build;
 
+  /// The maximum number of concurrent jobs.
+  ///
+  /// This currently only applies to the ninja build, passed as the -j
+  /// argument. If this field is left as the default value `0`, then the
+  /// concurrency level is determined automatically.
+  final int concurrency;
```

Comment:
Should we use a null default value instead of `0`? Just a thought, I don't feel strongly

Reply:
Either would work. I went with 0 in the interests of having fewer nulls in the universe.